src/jukebox/components/playermpd/__init__.py

```diff
@@ -363,7 +370,6 @@ def replay(self):
     @plugs.tag
     def toggle(self):
```

Comment:
Maybe change to "toggle_playback" or "toggle_playpause" to be more descriptive with the new "toggle_*" commands?

Reply:
Had the same feeling! :-)

Reply:
This would make sense but it does have an impact on existing Phonieboxes. This would be a breaking change for `cards.yaml` and it would require some updating logic. I would like to keep this separate from this PR.

Reply:
Will be addressed here #2201